Gopkg.toml

```diff
@@ -1,16 +1,14 @@
-# Stay exactly in step with a specific duffle release
-[[constraint]]
-  name = "github.com/deislabs/duffle"
-  version = "=0.2.0-beta.1"
-
 [[constraint]]
   name = "github.com/deislabs/cnab-go"
-  version = "0.2.0-beta.1"
+  branch = "master"
+  # version = "0.3.0-beta.1"
```

Comment:
Can we do a release of cnab-go today and then not rely on master? That doesn't need to hold up this PR, just asking if there's a reason we can't do another release.

Reply:
@carolynvs Yup, only intending to merge after this is updated with a new release 👍

Reply:
Version now updated to latest `v0.3.1-beta1`